Makefile

```diff
@@ -63,19 +52,19 @@ $(BUILD_DIR)/container-tag: $(BUILD_DIR)
 	echo -ne "$(IMAGE_TAG)" >$@
 
 unit-test: $(VENV)
-	$(VENV)/bin/python $(VENV)/bin/pip install -e$(SRCROOT) -e$(FIRKINROOT) \
+	$(VENV)/bin/python $(VENV)/bin/pip install -e$(SRCROOT) \
 	-e$(VAULTROOT) nose && \
 	$(VENV)/bin/nosetests -vd .
 
-$(VENV)/bin/y2j: $(VENV)
-	$(VENV)/bin/python $(VENV)/bin/pip install -e$(FIRKINROOT)
+y2j: $(VENV)
+	$(VENV)/bin/python $(VENV)/bin/pip install -e$(SRCROOT)
```

Comment:
Why is this used to setup y2j?

Reply:
Actually, you are directly using the y2j script. There is no need to even set it up in that case. I would just drop all dependency on y2j in this Makefile.

`$(VENV)/bin/python ./y2j $(STAGE)/app_metadata.yaml`